<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hideshowmat</class>
 <widget class="QDialog" name="hideshowmat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Materials to be Shown or Hidden...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Shown Materials</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="showlistWidget">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showall_pb">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Show all materials</string>
           </property>
           <property name="text">
            <string>Show All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="hide_pb">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Hide selected material</string>
           </property>
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="show_pb">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Show Selected Material</string>
           </property>
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Hidden Materials</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="hidelistWidget">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hideall_pb">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Hide all materials</string>
           </property>
           <property name="text">
            <string>Hide All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>hideshowmat</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>hideshowmat</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hide_pb</sender>
   <signal>clicked()</signal>
   <receiver>hideshowmat</receiver>
   <slot>hide_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_pb</sender>
   <signal>clicked()</signal>
   <receiver>hideshowmat</receiver>
   <slot>show_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hideall_pb</sender>
   <signal>clicked()</signal>
   <receiver>hideshowmat</receiver>
   <slot>hide_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showall_pb</sender>
   <signal>clicked()</signal>
   <receiver>hideshowmat</receiver>
   <slot>show_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>hide_all()</slot>
  <slot>show_all()</slot>
  <slot>hide_selected()</slot>
  <slot>show_selected()</slot>
 </slots>
</ui>
